doc/sphinx/user/methods/freesurface/arbitrary-le-implementation.md

```diff
@@ -21,21 +21,17 @@ the mesh velocity. The mesh velocity is calculated by solving
 -\Delta \textbf{u}_m &= 0 & \qquad & \textrm{in } \Omega, \\
 \textbf{u}_m &= \left( \textbf{u} \cdot \textbf{n} \right) \textbf{n} & \qquad & \textrm{on } \partial \Omega_{\textrm{free surface}}, \\
 \textbf{u}_m \cdot \textbf{n} &= 0 & \qquad & \textrm{on } \partial \Omega_{\textrm{free slip}}, \\
-\textbf{u}_m &= 0 & \qquad & \textrm{on } \partial \Omega_{\textrm{Dirichlet}}.\end{align}
+\textbf{u}_m &= 0 & \qquad & \textrm{on } \partial \Omega_{\textrm{Dirichlet}}.
+\end{align}
 ```
 After this mesh velocity is calculated, the mesh vertices are time-stepped
 explicitly. This scheme has the effect of choosing a minimally distorting
 perturbation to the mesh. Because the mesh velocity is no longer zero in the
 ALE approach, we must then correct the Eulerian advection terms in the
-advection system with the mesh velocity (see, e.g. (Donea et al. 2004)). For
-instance, the temperature equation {math:numref}`eq:temperature-Boussinesq-linear`27]
-becomes
+advection system with the mesh velocity (see, e.g., {cite}`donea:etal:2004`).
+For instance, the temperature equation
+{math:numref}`eq:temperature-Boussinesq-linear` becomes
 
 ```{math}
-\rho C_p \left(\frac{\partial T}{\partial t} + \left(\mathbf u - \mathbf u_m \right) \cdot\nabla T\right)
-  - \nabla\cdot k\nabla T
-  =
-  \rho H
-   \quad
-   \textrm{in $\Omega$}.
+\rho C_p \left(\frac{\partial T}{\partial t} + \left(\mathbf u - \mathbf u_m \right) \cdot\nabla T\right)- \nabla\cdot k\nabla T=  \rho H   \quad   \textrm{in $\Omega$}.
```

Comment:
Can you revert this change? I found the old version much easier to read (plus the new one has some weird spacing for some operators).